src/server/handlers/launchpad.js

```diff
@@ -400,7 +402,8 @@ export const findSnaps = async (req, res) => {
       payload: {
         code: 'snaps-found',
         snaps: snaps
-      }
+      },
+      ...normalize(snaps, snapList)
```

Comment:
Line of comment in the code would be useful to anyone doing the snaps refactoring later.